### PR TITLE
Features sb 25 sign in fe

### PR DESCRIPTION
**Pull request by ValentinaTenus**
1. create login form with logic in useSignInForm hook
2.  add validation for email and password
3. add api for signIn, save data after success and show error if fail
4. split saving user and tokens in state according to types that server returns
5. add query for getting user
6. change getting new access token according to changes on server
7. change style for auth page to make it more responsive(in height)
image
image
image
image

### CODE REVIEW

File: src/components/auth/sign-in-form/sign-in-form.tsx
Comment (lyudad) [inline src/components/auth/sign-in-form/sign-in-form.tsx:1]: I think you make too complicated structure by making folders for only one file

File: src/redux/user/validation/sign-in-schema.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import * as Yup from "yup";
+
+import { emailRegex, userValidationMessage } from "./constants/index.ts";
+
+const userSignInValidation = Yup.object().shape({
```
Comment (lyudad) [inline src/redux/user/validation/sign-in-schema.ts]: why its in redux? not near the component where you use this validation

Comment (ValentinaTenus) [reply src/redux/user/validation/sign-in-schema.ts]: I've changed it

File: src/components/auth/sign-in-form/sign-in-form.tsx
Comment (ValentinaTenus) [reply src/components/auth/sign-in-form/sign-in-form.tsx:1]: I've changed it.

File: src/components/example-form/example-form.tsx
Comment (lyudad) [inline src/components/example-form/example-form.tsx:1]: I assume it could be deleted

File: src/libs/hooks/use-get-user.ts
Comment (lyudad) [inline src/libs/hooks/use-get-user.ts:1]: hook file too far from the component file, its will be uncomfortable in the future. I don't think that you will use this hook from multiple components

File: src/libs/types/user/user-sign-in-request-dto.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+type UserSignInRequestDto = {
+	email: string;
+	password: string;
+};
```
Comment (lyudad) [inline src/libs/types/user/user-sign-in-request-dto.ts]: why we need separate file for one type? do you plan add more here?

File: src/pages/auth/index.tsx
Comment (lyudad) [inline src/pages/auth/index.tsx:1]: why libs folder needed in auth page? there no other folder in auth folder except libs, you just ad extra complexity. and why name is libs, components are not libraries, its just components

File: src/pages/auth/constants.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const exceptionMessage = {
+	EMAIL_CONFIRM: "Please, confirm email.",
```
Comment (VladManukian) [inline src/pages/auth/constants.ts:2]: I'm not sure, but it might break 1.3.3.

**Pull request merged**